src/python_testing/TC_OpstateCommon.py

```diff
@@ -117,6 +117,43 @@ def write_to_app_pipe(self, command):
         with open(self.app_pipe, "w") as app_pipe:
             app_pipe.write(command + "\n")
 
+    async def get_attributes(self, cluster, endpoint: int):
```

Comment:
Can you remove this? A dict of string -> int is more fragile that what's already code generated. You can get rid of all this code and just do ex. 

start_cmd_id = commands.StartCommand.id.

In fact, you don't even need to pull those into separate attributes - you can just use them from the codegen directly.

Reply:
Hi Cecille, 
I have removed the variable functions created in OpstateCommon module and followed the method you mentioned above and called the variable names directly in the if checks now in tests contained in OpstateCommon and RVCOPSTATE_2_1 test module. This should reduce the amount of clutter for variable setting that was in the tests previously.